frontend/src/components/SignUpSuccessDialog/index.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import { styles } from "./styles";
+
+export default function SuccessSignInDialog(isSignupSuccess) {
+  const [open, setOpen] = useState(true);
+  const navigate = useNavigate();
+
+  const handleCloseDialog = () => {
```

Comment:
Since this is working already, can remove` console.log`s and `navigate("#", { replace: true });` from `useSignup::handleSignUpSuccess`